tsdb/internal/meta.pb.go

```diff
@@ -16,7 +16,7 @@ It has these top-level messages:
 */
 package internal
 
-import proto "github.com/golang/protobuf/proto"
```

Comment:
Don't think changes to this file were intended.  Also, you might be using a different protobuf plugin.  Did you setup your env according to: https://github.com/influxdb/influxdb/blob/master/CONTRIBUTING.md#generated-google-protobuf-code